exo-run/src/service-restarter.ls

```diff
@@ -30,19 +30,21 @@ class ServiceRestarter extends EventEmitter
 
   _restart: ->
     @watcher.close!
-    cwd = @docker-config-location 
-    DockerCompose.kill-container {service-name: @role, cwd, @write}, (exit-code) ~>
-      | exit-code => @emit 'error', "Docker failed to kill container #{@role}"
-      @write "Docker container stopped"
-
-      DockerCompose.create-new-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
-        | exit-code => @emit 'error', "Docker image failed to rebuild #{@role}"
-        @write "Docker image rebuilt"
-
-        DockerCompose.start-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
-          | exit-code => @emit 'error', "Docker container failed to restart #{@role}"
-          @watch!
-          @logger.log {role: 'exo-run', text: "'#{@role}' restarted successfully"}
+    set-timeout ( ~>
+      cwd = @docker-config-location 
+      DockerCompose.kill-container {service-name: @role, cwd, @write}, (exit-code) ~>
+        | exit-code => @emit 'error', "Docker failed to kill container #{@role}"
+        @write "Docker container stopped"
+
+        DockerCompose.create-new-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
+          | exit-code => @emit 'error', "Docker image failed to rebuild #{@role}"
+          @write "Docker image rebuilt"
+
+          DockerCompose.start-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
+            | exit-code => @emit 'error', "Docker container failed to restart #{@role}"
+            @watch!
+            @logger.log {role: 'exo-run', text: "'#{@role}' restarted successfully"})
+    , 2500
```

Comment:
We can start with 1 second for this timeout and for the `stabilityThreshold` above? Yeah these aren't meant to be the same value, but @ben-goldberg you're the one who's actually been using the sdk the most, let us know what you think is an appropriate wait time.